ofl/notoserifolduyghur/METADATA.pb

```diff
@@ -0,0 +1,24 @@
+name: "Noto Serif Old Uyghur"
+designer: "Google"
+license: "OFL"
+category: "SERIF"
+date_added: "2023-09-26"
+fonts {
+  name: "Noto Serif Old Uyghur"
+  style: "normal"
+  weight: 400
+  filename: "NotoSerifOldUyghur-Regular.ttf"
+  post_script_name: "NotoSerifOldUyghur-Regular"
+  full_name: "Noto Serif Old Uyghur Regular"
+  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/old-uyghur)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "old-uyghur"
+source {
+  repository_url: "https://github.com/notofonts/old-uyghur.git"
+  archive_url: "https://github.com/notofonts/old-uyghur/releases/download/NotoSerifOldUyghur-v1.003/NotoSerifOldUyghur-v1.003.zip"
+}
+is_noto: true
+primary_script: "Ougr"
```

Comment:
@simoncozens are there now language data to add here?

Reply:
[Should be](https://github.com/googlefonts/lang/blob/main/Lib/gflanguages/data/languages/oui_Ougr.textproto).